src/config.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+
+export default class Config {
+    public readonly token: string;
+
+    public readonly prefix: string;
+
+    public whitelist: string[];
+
+    public emojis: string[];
+
+    constructor() {
+      this.token = '';
+      this.prefix = '';
+      this.whitelist = [''];
+      this.emojis = [''];
+    }
+
+    public static getConfig(): Config {
+      const data = fs.readFileSync('./config.yml');
```

Comment:
we should put './config.yml' in a constant in this file so that we don't need to change it in two different locations